{"name": "# C5188 Device Dashboard page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document\n  (Session info: chrome=86.0.4240.75)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 56, in dvc_is_on_the_pg\n    context.app.main_page.dvc_is_on_the_pg()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1053, in dvc_is_on_the_pg\n    actual_text = wait.until(EC.visibility_of_element_located(IMEI_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 76, in text\n    return self._execute(Command.GET_ELEMENT_TEXT)['value']\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1602901901893, "stop": 1602901904614}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1602901904615, "stop": 1602901907575}, {"name": "And Go to the Devices page https://devcloud.connectedio.com/devices", "status": "passed", "start": 1602901907575, "stop": 1602901907834}, {"name": "Then Click Add new device button on the right up corner", "status": "passed", "start": 1602901907835, "stop": 1602901908183}, {"name": "Then Fill required Device IMEI field 356961071557824", "status": "passed", "start": 1602901908183, "stop": 1602901908393}, {"name": "And Fill Device Name field if any TVM Test123 jjnkjn", "status": "passed", "start": 1602901908393, "stop": 1602901908596}, {"name": "Then Fill required Serial Number field 1806208A0279", "status": "passed", "start": 1602901908596, "stop": 1602901908755}, {"name": "And Click Save and refresh the page", "status": "passed", "start": 1602901908755, "stop": 1602901910197}, {"name": "Then Success is here - device is on the page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: element is not attached to the page document\n  (Session info: chrome=86.0.4240.75)\n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 56, in dvc_is_on_the_pg\n    context.app.main_page.dvc_is_on_the_pg()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1053, in dvc_is_on_the_pg\n    actual_text = wait.until(EC.visibility_of_element_located(IMEI_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 76, in text\n    return self._execute(Command.GET_ELEMENT_TEXT)['value']\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 633, in _execute\n    return self._parent.execute(command, params)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1602901910197, "stop": 1602901911096}, {"name": "Then Choose any device with IMEI number which is highlighted in blue active IMEI number and click on an active IMEI number", "status": "skipped", "start": 1602901913239, "stop": 1602901913239}, {"name": "Then Verify the Device Management Portal has opened", "status": "skipped", "start": 1602901913239, "stop": 1602901913239}], "start": 1602901896575, "stop": 1602901913239, "uuid": "2468b75c-71de-4d19-b29e-8e7058ce4226", "historyId": "06f5543c6cdade6c93c93ff17d488c0a", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5188 Device Dashboard page"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}